Build fails because of `autoprefixer` update

```
-----> Build
       Running build
       
       > pattern-recognition-server@0.1.1 build /tmp/build_e2daa028_
       > webpack --mode production
       
       Hash: a12568837194b509c258
       Version: webpack 4.44.2
       Time: 10565ms
       Built at: 09/19/2020 6:52:39 PM
        1 asset
       Entrypoint main = main.js
       [17] ./node_modules/dom-helpers/esm/css.js + 5 modules 1.82 KiB {0} [built]
            |    6 modules
       [21] ./node_modules/uncontrollable/esm/index.js + 4 modules 16.1 KiB {0} [built]
            |    5 modules
       [22] ./node_modules/@restart/hooks/esm/useEventCallback.js + 1 modules 831 bytes {0} [built]
            |    2 modules
       [30] ./node_modules/dom-helpers/esm/listen.js + 2 modules 1.91 KiB {0} [built]
            |    3 modules
       [32] (webpack)/buildin/global.js 472 bytes {0} [built]
       [45] ./src/styles/main.css 628 bytes {0} [built]
       [47] ./node_modules/mini-css-extract-plugin/dist/loader.js!./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./src/styles/main.css 1 KiB {0} [built] [failed] [1 error]
       [64] ./src/components/index.jsx + 63 modules 234 KiB {0} [built]
            | ./src/scripts/AidSprite.js 7.11 KiB [built]
            | ./src/scripts/AimSprite.js 9.12 KiB [built]
            | ./src/scripts/HelicopterSprite.js 8.45 KiB [built]
            | ./src/scripts/drawMatrix.js 2.69 KiB [built]
            | ./src/scripts/AnimationSprite.js 4.65 KiB [built]
            |     + 59 hidden modules
           + 215 hidden modules
       
       ERROR in ./src/styles/main.css (./node_modules/mini-css-extract-plugin/dist/loader.js!./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./src/styles/main.css)
       Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
       ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
       Error: true is not a PostCSS plugin
           at Processor.normalize (/tmp/build_e2daa028_/node_modules/postcss/lib/processor.js:168:15)
           at new Processor (/tmp/build_e2daa028_/node_modules/postcss/lib/processor.js:52:25)
           at postcss (/tmp/build_e2daa028_/node_modules/postcss/lib/postcss.js:55:10)
           at Object.loader (/tmp/build_e2daa028_/node_modules/postcss-loader/dist/index.js:94:41)
           at /tmp/build_e2daa028_/node_modules/webpack/lib/NormalModule.js:316:20
           at /tmp/build_e2daa028_/node_modules/loader-runner/lib/LoaderRunner.js:367:11
           at /tmp/build_e2daa028_/node_modules/loader-runner/lib/LoaderRunner.js:233:18
           at context.callback (/tmp/build_e2daa028_/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
           at Object.loader (/tmp/build_e2daa028_/node_modules/postcss-loader/dist/index.js:103:7)
        @ ./src/styles/main.css 2:26-195
        @ ./src/components/index.jsx
       Child HtmlWebpackCompiler:
            1 asset
           Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
           [0] ./node_modules/html-webpack-plugin/lib/loader.js!./src/views/index.html 1.11 KiB {0} [built]
       Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!node_modules/postcss-loader/dist/cjs.js!src/styles/main.css:
           Entrypoint mini-css-extract-plugin = *
           [0] ./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./src/styles/main.css 485 bytes {0} [built] [failed] [1 error]
           
           ERROR in ./src/styles/main.css (./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./src/styles/main.css)
           Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
           Error: true is not a PostCSS plugin
               at Processor.normalize (/tmp/build_e2daa028_/node_modules/postcss/lib/processor.js:168:15)
               at new Processor (/tmp/build_e2daa028_/node_modules/postcss/lib/processor.js:52:25)
               at postcss (/tmp/build_e2daa028_/node_modules/postcss/lib/postcss.js:55:10)
               at Object.loader (/tmp/build_e2daa028_/node_modules/postcss-loader/dist/index.js:94:41)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! pattern-recognition-server@0.1.1 build: `webpack --mode production`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the pattern-recognition-server@0.1.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /tmp/npmcache.wcEiT/_logs/2020-09-19T18_52_39_846Z-debug.log
-----> Build failed
```

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -13,6 +13,11 @@ and this project adheres to `Semantic Versioning`_.
 Unreleased_
 ===========
 
+Fixed
+-----
+
+- Build failed with new ``autoprefixer`` (version ``10``).
+
 `0.2.0`_ - 2020-09-19
 =====================
 

--- a/package.json
+++ b/package.json
@@ -64,7 +64,7 @@
     "@babel/plugin-transform-runtime": "^7.11.5",
     "@babel/preset-env": "^7.11.5",
     "@babel/preset-react": "^7.10.4",
-    "autoprefixer": "^10.0.0",
+    "autoprefixer": "^9.0.0",
     "babel-eslint": "^10.1.0",
     "babel-loader": "^8.1.0",
     "css-loader": "^4.3.0",

